### DISCUSSION TITLE
Plasticity Parameters

### DISCUSSION THREAD

**Discussion by siwux** [neperfepx/FEPX#93] [Q&A] [answered]
Hi There,
I'm going to use NEPER and FEPX to study the crystal plasticity behaviour of the AlMg alloy.I don't know what experiment to use to get these relevant parameters.Could you please give me some advice?Thanks in advance.

**Comment by mkasemer** [comment, marked as answer]
@siwux : This is actually a very complicated question. There's somewhat of a decision tree that you'll have to follow to decide the best path.

- First, what material are you studying specifically? Is it a well-studied material (e.g., 6061), or a new alloy?
  - If the former, you can likely find parameter sets described in literature, if others have performed CPFEM modeling with the alloy before.
  - If the latter, how much different is it than an existing alloy? Can you justify that its behavior is close to an existing, well-studied alloy, and thus can use those parameters?
- If you must determine your parameters from experiments, then this can quickly become a very difficult problem. Potentially you can fit your parameters via macroscopic stress-strain curves, but the solution is probably non-unique: i.e., a number of parameter sets will allow you to match the macroscopic behavior. Again, if you can find a good starting point from literature, you may be able to reduce the chances of finding an erroneous parameter set, or by fitting to stress-strain curves from samples cut in many directions, but this is also cumbersome.
- Micromechanical tests (EBSD+DIC, HEXRD) are the most sophisticated ways to estimate parameter sets, though these methods are very time consuming and require specific skillsets, and may necessitate some extra training if you are not well-versed in these methods.
- You may need to consider some secondary effects as well: does your material contain precipitates / is it precipitate hardened? If so, are you going to investigate only a single micromechanical state, or will you be looking at the behavior pre-aged, post-aged, heat treated, etc.? If the latter, it may necessitate even further testing, literature review, and model implementation to consider precipitates properly.

I hope this helps.

**Reply by siwux** [reply]
Okay! many thanks for your help!
